tests/integration/components/phone-input-test.js

```diff
@@ -128,4 +128,20 @@ module('Integration | Component | phone-input', function(hooks) {
     );
     assert.ok(find('input').disabled);
   });
+
+  test('can prevent the dropdown', async function(assert) {
+    assert.expect(1);
+
+    await this.owner.lookup('service:phone-input').load();
```

Comment:
The library is loaded in a `beforeEach` hook: https://github.com/qonto/ember-phone-input/pull/134/files#diff-0e2859b8508db13adaa5918464d0aeecR10

You can safely remove this line.